codes/quantum/qudits_galois/galois_polynomial.yml

```diff
@@ -9,14 +9,25 @@ logical: galois
 
 name: 'Galois-qudit polynomial code'
 introduced: '\cite{arxiv:quant-ph/9906129}'
-# Gottesman's book
 
-description: 'Stub.'
 
-protection: 'Also called \textit{quantum Reed-Solomon code}.'
+description: 'Let \(C_1\) be a Reed-Solomon code with \([n,k_1,d_1]_q\) and \(C_2^\perp\) with \([n,k_2,d_2]_q\) to be a modified Reed-Solomon code such that \(C_2^\perp \subseteq C_1\)
```

Comment:
Notation I use for Galois-qudit codes is [[n,k,d]]_{GF(q)} to differentiate from cyclic-qudit [[n,k,d]]_q codes.

Reply:
I replaced `_q` to ` _{\mathrm{GF}(q)}`